package.json

```diff
@@ -8,7 +8,8 @@
     "lint": "eslint .",
     "lintfix": "eslint --fix lib/**",
     "test": "eslint lib/** && jest --coverage --maxWorkers=4",
-    "coveralls": "jest --coverage && cat ./tests/coverage/lcov.info | coveralls"
+    "coveralls": "jest --coverage && cat ./tests/coverage/lcov.info | coveralls",
+    "doc": "jsdoc -r lib -d docs/generated -R README.md"
```

Comment:
it worth add `jsdoc` to dependencies list. Currently it would work only with global jsdoc

Reply:
I agree with @vvscode, jasdoc should be a dev dependency.